### ISSUE TITLE
Reflection warnings

### ISSUE THREAD

**Issue by dchelimsky** [cognitect-labs/aws-api#166]
## Dependencies

Be sure to list the precise libs and versions you are using ("the
latest" might change by the time we're looking at your issue).

e.g.

``` clojure
{:deps {com.cognitect.aws/api       {:mvn/version "0.8.484"}
        com.cognitect.aws/endpoints {:mvn/version "1.1.11.926"}
        com.cognitect.aws/s3        {:mvn/version "810.2.817.0"}}}
```

## Description with failing test case

Per [this comment](https://github.com/cognitect-labs/aws-api/issues/120#issuecomment-753605945), there are a number of reflection warnings that we need to address.

**Comment by dchelimsky** [comment]
Addressed in b65d61d6e69880786061d8c7ad5893cce8bc14a8 and 515daf9273edeb16df8ee158cfb90cef5c3d30c2

**Comment by dchelimsky** [comment]
Released in aws-api 0.8.498
